openwebstart/src/main/java/com/openwebstart/jvm/RemoteRuntimeManager.java

```diff
@@ -44,6 +45,16 @@ private RemoteRuntimeManager() {
 
         LOG.debug("Trying to find remote Java runtime. Requested version: '{}' Requested vendor: '{}' requested os: '{}'", versionString, vendor, operationSystem);
 
+        // jvm server specified in jnlp is not null and is allowed
+        // TODO : What to do if not found in Whitelist
+        // TODO : confirm that we only check the URL in JNLP against whitelist and not the URL specified in settings
+        if (RuntimeManagerConfig.isNonDefaultServerAllowed() && specificServerEndpoint != null) {
+            if (!UrlWhiteListUtils.isUrlInWhitelist(specificServerEndpoint,  RuntimeManagerConfig.getJvmServerWhitelist())) {
+                LOG.warn("*************** JVM Server URL {} not in JVM Server Whitelist", specificServerEndpoint);
```

Comment:
```suggestion
                LOG.warn("JVM Server URL {} not in JVM Server Whitelist", specificServerEndpoint);
```

Reply:
this was only while developing to spot the msg in log easily without strinung eyes. will be deleted before in final version. But we need to discuss the two TODOs - 1. Ok rertuen Optionla.empty if not fin whitelist? 2. Only the jvm server specified in jnlp has to be checked against whitleist. If allow jvm from jnlp is true an no jvm url is specified in jnlp then it falls back to the jvm url specified in the settings. In this case should that jvm url be checked against whitelist?